src/App.tsx

```diff
@@ -1,10 +1,19 @@
 import React from "react";
-import Landing from "@landing";
+import { Landing } from "./pages";
+import { SignIn, About } from "@components/index";
+import { Routes, Route } from "react-router-dom";
 
 const App = () => {
   return (
     /*Private routing will be added once main screen is done*/
-    <Landing />
+    <>
```

Comment:
WDYT to use implicit return here? Also, I think there's not need to wrap `<Routes>` with `<>`

Reply:
I agree, I'd added the implicit import and taken out the fragment that shouldnt be there